index.php

```diff
@@ -20,7 +20,9 @@
 require_once 'boot.php';
 require_once 'object/BaseObject.php';
 
-$a = new App(__DIR__);
+if (empty($a)) {
```

Comment:
Again the same code as in `get_app()`! Encapsulate this, please.

Reply:
Yeah, could be done, I guess.